### ISSUE TITLE
Process 'Percolator' finished, exit code: -1073741819 in Non-specific HLA DIA workflow

### ISSUE THREAD

**Issue by marthanik** [Nesvilab/FragPipe#1949]
Hello!

We are new to fragpipe and tried to use the non-specific HLA DIA workflow for immunopeptidomics with both DDA and DIA files from a Thermo Q Exactive HF-X instrument and we got the following error that terminated the search: 

"Found 0 test set PSMs with q<0.01.
No targets found with q<0.01
Resetting score vector, using default vector. Use --override flag to prevent this.
Split 1:	Selected feature 1 as initial direction. Could separate 2 training set positives with q<0.01 in that direction.
Split 2:	Selected feature 1 as initial direction. Could separate 2 training set positives with q<0.01 in that direction.
Split 3:	Selected feature 1 as initial direction. Could separate 2 training set positives with q<0.01 in that direction.
Selected best-scoring PSM per scan+expMass (target-decoy competition): 3 target PSMs and 0 decoy PSMs.
Calculating q values.
Final list yields 0 target PSMs with q<0.01.
Calculating posterior error probabilities (PEPs).
ERROR: Only 1 bin available for PEP estimation, no distinguishing feature present.
No-terminate flag set: ignoring error and adding random noise to scores for PEP calculation.
Logit transforming all scores prior to PEP calculation
Process 'Percolator' finished, exit code: -1073741819
Process returned non-zero exit code, stopping

The detailed log file is also uploaded with more detailed info.

Any help on resolving this would be appreciated!

Best regards,
Martha

[log_2024-12-18_15-31-17.zip](https://github.com/user-attachments/files/18183650/log_2024-12-18_15-31-17.zip)

**Comment by dpolasky** [comment]
Hi Martha,
It looks like one of the files extracted by DIA-Umpire from your blank run had no PSMs, which caused the pipeline to stop. Given that it was a blank, I guess it is probably expected that there are no results to extract - removing that file from the search should resolve the issue.

Best,
Dan

**Comment by fcyu** [comment]
@marthanik This is a know issue and has been fixed in FragPipe 21.0: https://github.com/Nesvilab/FragPipe/issues/1374#issuecomment-1863520255. Please upgrade it to the latest version and try again.

Best,

Fengchao
